src/about.js

```diff
@@ -43,6 +43,7 @@ ${getBlueprintVersion()}
       "Sonny Piers https://sonny.re",
       "Lorenz Wildberg https://gitlab.gnome.org/lwildberg",
       "Andy Holmes https://gitlab.gnome.org/andyholmes",
+      "Alan Beveridge https://github.com/ADBeveridge",
```

Comment:
I'm in a back log of stuff that I'm doing for work at the present at work, so for now, i'll say no. I'll just work on having Workbench compile and integrate reasonably with C for now, since its a reasonable task. Sorry I didn't get the notification before, I was exploring GitHub's new UI, and found the Inbox tab. And this.

Reply:
I see. I can become C maintainer then.  You'll have to ping me when I have some work, as I won't be checking around Workbench's issues on my own. Sound good?

Reply:
You should get auto-pinged if you become code owner, so that wouldn't be a problem.